### ISSUE TITLE
Ansible 2.7 emits DEPRECATION WARNING on 'Purge PHP version packages' task

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/ansible-role-php-versions#24]
From a recent Drupal VM build on Ansible 2.7.0:

```
TASK [geerlingguy.php-versions : Purge PHP version packages.] ******************
[DEPRECATION WARNING]: Invoking "apt" only once while using a loop via 
squash_actions is deprecated. Instead of using a loop to supply multiple items 
and specifying `name: {{ item }}`, please use `name: [u'php5.6-common', 
u'php7.0-common', u'php7.1-common', u'php7.2-common']` and remove the loop. 
This feature will be removed in version 2.11. Deprecation warnings can be 
disabled by setting deprecation_warnings=False in ansible.cfg.
ok: [drupalvm] => (item=[u'php5.6-common', u'php7.0-common', u'php7.2-common'])
```

The problem here is, the `with_items` loop is not necessary for the apt name parameter, but it _is_ used as part of the `when` clause to identify which items the `apt` module should operate on...

Here's the full task:

```yaml
# PHP package purges.
- name: Purge PHP version packages.
  apt:
    name: "{{ item }}"
    state: absent
    purge: true
    force: true
  with_items:
    - php5.6-common
    - php7.0-common
    - php7.1-common
    - php7.2-common
  when: "'php' + php_version not in item"
```

**Comment by geerlingguy** [comment]
So I'm trying to figure out the best way to approach this. Maybe I'll have to do a `set_fact` on a loop or something?

**Comment by benjifisher** [comment]
I ran into this deprecation notice while setting up DrupalVM on my home machine, so I thought I would lend a hand: see PR #32.

**Comment by geerlingguy** [comment]
Fixed via #37 — totally missed #32, sorry about that!
